modules/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -24,13 +24,16 @@
 
 import {Layer, COORDINATE_SYSTEM} from '@deck.gl/core';
 import GL from '@luma.gl/constants';
-import {Model, Geometry, loadTextures, Texture2D, fp64} from 'luma.gl';
+import {Model, Geometry, loadTextures, Texture2D, fp64, Buffer} from 'luma.gl';
+import {Matrix4} from 'math.gl';
 const {fp64LowPart} = fp64;
 
 import vs from './mesh-layer-vertex.glsl';
 import fs from './mesh-layer-fragment.glsl';
 
 const RADIAN_PER_DEGREE = Math.PI / 180;
+let rotationMatrix = new Float32Array(16);
+let xformMatrix = new Matrix4();
```

Comment:
Nit: We called this `instanceModelMatrix` above. Have a preference for staying as consistent as possible with naming, at the expense of a few more characters.